src/model/package.entity.ts

```diff
@@ -0,0 +1,24 @@
+import { Entity, PrimaryKey, Property, Collection, OneToMany } from '@mikro-orm/core';
+import { Version } from './version.entity';
+import { Download } from './download.entity';
+
+@Entity()
+export class Package {
+  @PrimaryKey()
+  id!: number;
+
+  @Property()
+  name!: string;
```

Comment:
make it primary key. It will simplify a lot, and we cannot have duplicates by name anyway